waspc/data/Generator/templates/db/schema.prisma

```diff
@@ -7,6 +7,7 @@ datasource db {
 
 generator client {
   provider = "prisma-client-js"
+  output = "../server/node_modules/.prisma/client"
```

Comment:
Aha, ok let's just keep in mind this is hardcoded, ideally Generator would generate this path based on its knowledge of where server is located. But ok I don't think it is worth spending much time on this at the moment.

Reply:
ah yes true, forgot about that! will try to add TODO comment.